app/views/ServiceInfoPage.scala.html

```diff
@@ -85,6 +86,14 @@ <h1 id="service-header" class="page-heading mt-4 position-relative">
       </div>
     }
 
+    @if(lifecycleStatus.contains(DecommissionInProgress)) {
+        <div class="alert alert-warning">
+            <strong>
+                This service is marked for decommissioning, see <a id="decommission-state" href="/service/@{repositoryDetails.name}/commissioning-state">commissioning state</a> for progress. If this is unexpected contact <a href="https://hmrcdigital.slack.com/messages/team-platops" target=\"_blank\" rel=\"noreferrer noopener\">#team-platops<span class="glyphicon glyphicon-new-window"/></a> for help.
```

Comment:
can you use the reverse route for the url?

Reply:
Done